app/store/constants.js

```diff
@@ -0,0 +1,13 @@
+export default {
```

Comment:
I've been reading a little bit and there's a cleaner and safer way to define string constants using [Symbol](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol), so that the ActionType value cannot be changed, interpolated or used in combination with other values. It can only be used as it is.

FIREBASE_INIT: Symbol('FIREBASE_INIT')

Reply:
Good! I'll do it

Reply:
I can't use it, because vuex doesn't have support to Symbol.
Throw this error:

Reply:
Alright! We can just leave constants as String type.